blocks/library/cover-image/index.js

```diff
@@ -177,3 +183,12 @@ registerBlockType( 'core/cover-image', {
 		);
 	},
 } );
+
+function dimRatioToClass( ratio ) {
+	let rounded = Math.round( ratio / 10 );
+	// String#padStart not ready for prime time
+	if ( rounded < 10 ) {
+		rounded = '0' + rounded;
+	}
+	return 'has-background-dim--' + rounded;
```

Comment:
What was the reasoning behind the double dash? Could be misleading as a pattern that we don't encourage. Should be sufficient with the single dash.

Reply:
No reasoning, just reflexes.